docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,11 @@ Version history
 
 1.11.0 (Pending)
 ================
+* hot restart: stats are no longer shared between hot restart parent/child via shared memory, but rather by RPC.
+* server: changed protocol that hot restart children/parents use; now protobuf based.
```

Comment:
nit: I would remove this line from release notes, I don't think end users care about this though it is neat. ;)

Reply:
Ahhh, yes, wasn't aware of the end-user focus of this.